### PR TITLE
Create inline css

### PR DESCRIPTION
**Pull request by digitalsadhu**
This PR introduces a new constructed stylesheet.

This constructed stylesheet is created using PostCSS and is modelled after how Fabric CSS is produced in @fabric-ds/css which some notable distinctions which are:
1. The classes in @fabric-ds/css/component-classes are used as input into Tailwind purge's "safelist" to ensure that the CSS required for these is always present.
2. The Fabric elements are used as input for a Tailwind purge operation to throw away all CSS not being used and thereby create a smaller footprint.

The current size of the file generated is about 40kb though this currently still includes the Fabric components.css file which can be removed once we remove our dependence on the classes it provides.

Some elements will need to be slightly rewritten to ensure purge works as promised so this PR only uses the new constructed stylesheet for the button element initially. We can take a component or three at a time and port them over as we get time.

### CODE REVIEW

**Review by pearofducks — APPROVED** [review]
Nice job!

File: build-elements.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import path from 'path';
+import postcss from 'postcss';
+import tailwind from 'tailwindcss';
+import autoprefixer from 'autoprefixer';
+import cssnano from 'cssnano';
+import atImport from 'postcss-import';
+import presetEnv from 'postcss-preset-env';
+import importSvg from 'postcss-import-svg';
+import { breakpoints as customMedia } from '@fabric-ds/css/colors.js';
+import tailwindConfig from '@fabric-ds/tailwind-config';
```
Comment (BalbinaK) [inline build-elements.js:13]: @digitalsadhu What's the difference between contents of`@fabric-ds/tailwind-config` and what's in `@fabric-ds/css/src/utils/tailwind`?

Comment (digitalsadhu) [reply build-elements.js:13]: A very good question...
* @fabric-ds/css/src/utils/tailwind is not (or should not be) available for direct usage via that path though it should be via @fabric-ds/css/tailwind-config. See package exports [here](https://github.com/fabric-ds/css/blob/01fd80a27461cf0d593911e974853f84963d0681/package.json#L13). 
* `@fabric-ds/tailwind-config` on the other hand.... appears to just be super outdated and not actually maintained anymore. We must have changed that at some point. Good catch, I'll fix

Comment (digitalsadhu) [reply build-elements.js:13]: See https://github.com/fabric-ds/elements/pull/124/commits/ddf531100ba78b78f5bedc02e20d6ec76db87212

**Pull request merged**